solr/core/src/java/org/apache/solr/handler/component/StatsValuesFactory.java

```diff
@@ -182,7 +182,7 @@ protected AbstractStatsValues(StatsField statsField) {
       // "NumericValueSourceStatsValues" which would have diff parent classes
       //
       // part of the complexity here being that the StatsValues API serves two
-      // masters: collecting concrete Values from things like DocValuesStats and
+      // primaries: collecting concrete Values from things like DocValuesStats and
```

Comment:
Interestingly, this has nothing to do with replication, no clue what this means here